docs/loading/Loading_data_template.md

```diff
@@ -0,0 +1,348 @@
+
+### Notes embedded in the template
+
+There are some notes in this template, they are in bold italic format, please delete them as you write:
+
+```markdown
+***Note: descriptive text***
+```
+
+## Finally, the start of the template
+
+***Note: If there are multiple recommended choices, tell the
+reader this in the intro. For example, when loading from S3,
+there is an option for synchronous loading, and asynchronous loading:***
+
+StarRocks provides two options for loading data from S3:
+
+1. Asynchronous loading using Broker Load
+2. Synchronous loading using the `FILES()` table function
+
+***Note: Tell the reader WHY they would choose one choice over the other:***
+
+Small datasets are often loaded synchronously using the `FILES()` table function, and large datasets are often loaded asynchronously using Broker Load. The two methods have different advantages and are described below.
+
+> Note
+>
+> You can load data into StarRocks tables only as a user who has the INSERT privilege on those StarRocks tables. If you do not have the INSERT privilege, follow the instructions provided in [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) to grant the INSERT privilege to the user that you use to connect to your StarRocks cluster.
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+***Note: Processes that involve multiple components or steps may be easier to understand with a diagram. This example includes a diagram that helps describe the steps that happen when a user chooses the Broker Load option.***
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
+
+Create a table, start a load process that pulls a Parquet file from S3, and verify the progress and success of the data loading.
+
+> Note
+>
+> The examples use a sample dataset in Parquet format, if you want to load a CSV or ORC file, that information is linked at the bottom of this page.
+
+#### Create a table
+
+Create a database for your table:
+
+```SQL
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+```
+
+Create a table. This schema matches a sample dataset that you can load from a StarRocks S3 bucket.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/d40fd910-0530-47d5-8857-3dee7a400a5f)
Typo? It should be "an AWS S3 bucket" I think.

Reply:
These are S3 buckets in the StarRocks account.   I could say "This schema matches a sample dataset in an S3 bucket hosted in a StarRocks account."  Is that clearer @amber-create ?